paddle/framework/lod_tensor.cc

```diff
@@ -103,5 +103,34 @@ void LoDTensor::ShrinkInLevel(size_t level, size_t elem_begin,
   lod_ = new_lod;
 }
 
+Vector<size_t> expand_lod(Vector<size_t> level, Vector<size_t> starts,
+                          Vector<size_t> scales, bool repeat) {
```

Comment:
Functions names:  "CamelCase"

https://google.github.io/styleguide/cppguide.html#Function_Names

Now, only the seq_expand needs this function, it may be removed to seq_expand_op.

Reply:
Fixed.